backend/routes/misc.py

```
@@ -197,4 +197,88 @@ def system_status():
             'status': 'error',
             'error': str(e),
             'timestamp': datetime.utcnow().isoformat()
-        }), 500
\ No newline at end of file
+        }), 500
+
+@misc_bp.route('/service-health', methods=['GET'])
+def service_health():
```

Pylint (3.3.7)

[refactor] Too many branches (17/12)

(R0912)